tap_dixa/client.py

```diff
@@ -170,9 +75,9 @@ def _make_request(self, url, method, headers=None, params=None, data=None) -> di
         :param data: The data passed to the body of the request
         :return: A dictionary representing the response from the API
         """
-
         with self._session as session:
-            response = session.request(method, url, headers=headers, params=params, data=data)
+            response = session.request(
+                method, url, headers=headers, params=params, data=data)
```

Comment:
Can we either format the function calls like this:
```suggestion
            response = session.request(method, url, headers=headers, params=params, data=data)
```
or this:
```suggestion
            response = session.request(method,
                                       url,
                                       headers=headers,
                                       params=params,
                                       data=data)
```